assets/js/blocks/price-filter/index.tsx

```diff
@@ -63,12 +64,14 @@ registerBlockType( metadata, {
 			showFilterButton,
 			heading,
 			headingLevel,
+			inlineInput,
 		} = attributes;
 		const data = {
 			'data-showinputfields': showInputFields,
 			'data-showfilterbutton': showFilterButton,
 			'data-heading': heading,
 			'data-heading-level': headingLevel,
+			'data-inlineinput': inlineInput,
```

Comment:
This will trigger the 'Block Recovery Prompt'. Do you think we can fix the rendering without affecting the current `Save` component?